src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/models/CCJContent.java

```diff
@@ -26,18 +26,21 @@ public CCJContent(Claim claim) {
         requireNonNull(claim);
 
         this.claimReferenceNumber = claim.getReferenceNumber();
-        TheirDetails defendant = claim.getClaimData().getDefendant();
+        CountyCourtJudgment countyCourtJudgment = claim.getCountyCourtJudgment();
+        TheirDetails defendant = countyCourtJudgment.getDefendant();
         this.defendantName = defendant.getName();
         this.defendantAddress = defendant.getAddress();
-        this.paymentRepaymentOption = claim.getCountyCourtJudgment().getPaymentOption().name();
+        this.paymentRepaymentOption = countyCourtJudgment.getPaymentOption().name();
```

Comment:
I think `repaymentOption` should be enough.

Reply:
? we want to display payment option as string

Reply:
I meant the field name.